server/pulp/server/webservices/views/content.py

```diff
@@ -167,7 +170,7 @@ def post(self, request):
 
         :raises: OperationPostponed when an async operation is performed
         """
-        all_orphans = request.body_as_json
+        all_orphans = request.body_as_json or {}
```

Comment:
This actually does work as a ternary assignment, and is considered idiomatic python I believe.

    In [1]: request = None

    In [2]: x = request or {}

    In [3]: x
    Out[3]: {}

A chain of `or`s will return the first truthy value, if none are truthy, it will return the last.

Likewise a chain of `and`s will return the last truthy value, if none are truthy it returns the first.

This works because the truthyness of the value stored in the variable will always match the boolean value of the conditional.